apps/omg_watcher/lib/db/eth_event.ex

```diff
@@ -74,9 +74,14 @@ defmodule OMG.Watcher.DB.EthEvent do
   """
   @spec insert_exits!([non_neg_integer()]) :: :ok
   def insert_exits!(exits) do
-    Enum.each(exits, &(&1 |> Utxo.Position.decode() |> insert_exit!()))
+    exits
+    |> Enum.map(&parse_contract_exit_event/1)
+    |> Enum.each(&insert_exit!/1)
   end
 
+  @spec parse_contract_exit_event(%{call_data: %{utxo_pos: pos_integer()}}) :: Utxo.Position.t()
```

Comment:
Not sure name is adequate - I thought `call_data` is not event... and since it's interested in one field maybe `get_utxo_pos` :question:

Reply:
`call_data` is what we call the "extension" of the event data taken from the transaction's call data. I'll clarfify the names